scripts/migrate-data-to-enums.js

```diff
@@ -0,0 +1,151 @@
+const {
```

Comment:
my main concern here is what will happen if either migration fails part way through, particularly with regard to Cosmos and running out of transaction units. We need to be able to track which records have been transformed (maybe a temporary table/collection) for each dataset, and also record errors. Then we can only reattempt the ones that haven't been transformed. I'm quite nervous about this migration so we need to cover ourselves I think.

Reply:
Have updated the migration with additional logging and storage of which records have been updated.  It can be rerun and will pick up where it left off if it fails.